src/Integration.Vsix/SonarLintTagger/IssueToFilterableIssueConverter.cs

```diff
@@ -56,8 +56,8 @@ public static IEnumerable<IFilterableIssue> Convert(IEnumerable<IAnalysisIssue>
                 return null;
             }
 
-            // A start line of zero means the issue is file-level i.e. not associated with a particular line
-            if (issue.StartLine == 0)
+            // A start line of zero or empty file path means the issue is file-level i.e. not associated with a particular line
+            if (issue.StartLine == 0  || string.IsNullOrEmpty(issue.FilePath))
```

Comment:
I don't know why, but the StartLine returned is not 0. Maybe it's a bug in the subprocess. In either case, I can also change the Protocol to set it to 0.

Reply:
I'd check with the CFamily folk and ask what the line numbers represent in this case.